src/context/block_unit.rs

```diff
@@ -689,7 +687,7 @@ impl<'a> ContextWriter<'a> {
     if bs.cfl_allowed() {
       symbol_with_update!(self, w, uv_mode as u32, cdf);
     } else {
-      symbol_with_update!(self, w, uv_mode as u32, &mut cdf[..UV_INTRA_MODES]);
+      symbol_with_update!(self, w, uv_mode as u32, &mut cdf[..INTRA_MODES]);
```

Comment:
This may seem a little odd. Where the original code had `UV_INTRA_MODES`, it ought to read `INTRA_MODES + 1` because the additional element here is the CDF counter rather than the chroma-from-luma symbol. As with other edits in this patch, we are subtracting 1 now that the trailing zero is removed. Thus `UV_INTRA_MODES` becomes `INTRA_MODES`.